src/app/routes/applications/new-role/components/role-field/role-field.component.html

```diff
@@ -92,18 +92,18 @@
                 <mat-label class="ml-3 pl-1">Min Date</mat-label>
                 <mat-form-field class="mt-2" appearance="outline" floatPlaceholder="never">
                   <input [matDatepicker]="pickerMin" matInput placeholder="Min Date" autocomplete="off"
-                         formControlName="minDate" (keypress)="$event.preventDefault()">
+                         formControlName="minDate" disabled>
```

Comment:
I'm not a designer so i'll ask our expert. @davycastillo what do you think from UX perspective about disabling ability to write a date by user? It is desired behavior?

Reply:
I think it's more convenient to allow the user to edit the date but, just need to add the error message accordingly.

Reply:
if you do this, then you need to add more than just an error message to a full-fledged validation with valid characters and formats